tests/modules/hub.py

```diff
@@ -186,34 +156,152 @@ def forward(self, x):
         x = self.conv1(x)
         return x
 
-
-conditional_model = FallbackIf().eval().cuda()
-conditional_script_model = torch.jit.script(conditional_model)
-torch.jit.save(conditional_script_model, "conditional_scripted.jit.pt")
-
-enc = BertTokenizer.from_pretrained("bert-base-uncased")
-text = "[CLS] Who was Jim Henson ? [SEP] Jim Henson was a puppeteer [SEP]"
-tokenized_text = enc.tokenize(text)
-masked_index = 8
-tokenized_text[masked_index] = "[MASK]"
-indexed_tokens = enc.convert_tokens_to_ids(tokenized_text)
-segments_ids = [0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1]
-tokens_tensor = torch.tensor([indexed_tokens])
-segments_tensors = torch.tensor([segments_ids])
-dummy_input = [tokens_tensor, segments_tensors]
-
-config = BertConfig(
-    vocab_size_or_config_json_file=32000,
-    hidden_size=768,
-    num_hidden_layers=12,
-    num_attention_heads=12,
-    intermediate_size=3072,
-    torchscript=True,
-)
-
-model = BertModel(config)
-model.eval()
-model = BertModel.from_pretrained("bert-base-uncased", torchscript=True)
-
-traced_model = torch.jit.trace(model, [tokens_tensor, segments_tensors])
-torch.jit.save(traced_model, "bert_base_uncased_traced.jit.pt")
+class ModelManifest:
```

Comment:
Why not just add these to the dictionary?

Reply:
Do we want to trace and script the local custom models on every run regardless of the model file present?
I am referring to model['custom_model']['model'] field

Reply:
You can just pick one or the other

Reply:
I don't follow. What will be in model['custom_model_x']['model'] field in this case?

Reply:
The path field in the dictionary controls if you get script, trace or both